jobs/models.py

```diff
@@ -69,7 +68,81 @@ def set_expiration_date(self):
                 self.expiration_date = self.published_date + timedelta(60)
                 self.save()
 
-
     def __unicode__(self):
         return "{0}, {1}".format(self.title, self.company)
 
+
+class Meetup(models.Model):
+
+    MEETUP = 'MEET'
+    CONFERENCE = 'CONF'
+    WORKSHOP = 'WORK'
+    MEETUP_TYPES = (
+        (MEETUP, 'meetup'),
+        (CONFERENCE, 'conference'),
+        (WORKSHOP, 'workshop'),
+    )
+
+    title = models.CharField(max_length=500)
+    organization = models.ForeignKey(
+        'Company',
+        related_name="meetups",
+        blank=True,
+        null=True,
+    )
+    type = models.CharField(max_length=4, choices=MEETUP_TYPES, default=MEETUP)
+    contact_email = models.EmailField(max_length=254)
+    city = models.CharField(max_length=100)
+    country = CountryField()
+    description = models.TextField(max_length=5000)
```

Comment:
I've just copied it from the Job model really. And if we don't need to set any limit, that's OK with me to remove it. Shall I update the Job model as well?